README.md

```diff
@@ -81,15 +81,15 @@ The camelCase D functions will be PascalCase in Excel.
 
 ## Python versions
 
-Since autowrap depends on PyD, the python version must be explicitly stated
-as a dub configuration and defaults to 3.6. To use another version, pass
-`-c $CONFIG` to dub where `$CONFIG` is one of:
-
-* python27
-* python34
-* python35
-* python36
-
+Since autowrap depends on PyD, the version of python to use must be specified.
+You can either specify the configuration explicity, e.g.
+`subConfiguration "autowrap:python" "python36"` or you can use the default `env`
+configuration. In order to build using the `env` configuration your must first
+set the relevant environment variables using the `pyd_set_env_vars.{sh,bat}`
```

Comment:
This needs to be done automatically, and in a temporary directory instead of polluting the wherever the user is working.

Reply:
How? You can't usefully source files in dub `preBuildCommand`s, it's run in a sub-process.